docs/api/app.md

```diff
@@ -701,16 +701,16 @@ starts:
 const {app} = require('electron')
 let myWindow = null
 
-const shouldQuit = app.makeSingleInstance((commandLine, workingDirectory) => {
+const isNotFirstInstance = app.makeSingleInstance((commandLine, workingDirectory) => {
```

Comment:
I like the clarification in the first paragraph but I disagree with this variable name change.  Personally although `shouldQuit` is not the best self-explaining variable name I think `isNotFirstInstance` is more confusing / less explanatory.

Maybe removing the negative would make it more clear?

```js
const isSecondInstance = app.makeSingleInstance(...
```